src/cli/backup/sharepoint_e2e_test.go

```diff
@@ -216,3 +490,23 @@ func (suite *BackupDeleteSharePointE2ESuite) TestSharePointBackupDeleteCmd_NoBac
 	err := cmd.ExecuteContext(ctx)
 	require.Error(t, err, clues.ToCore(err))
 }
+
+// ---------------------------------------------------------------------------
+// helpers
+// ---------------------------------------------------------------------------
+
+func buildSharepointBackupCmd(
+	ctx context.Context,
+	configFile, site, category string,
+	recorder *strings.Builder,
+) (*cobra.Command, context.Context) {
+	cmd := cliTD.StubRootCmd(
+		"backup", "create", "sharepoint",
+		"--config-file", configFile,
```

Comment:
does `config-file` not have a flags entry?  if it doesn't, could you please add a tech debt ticket to fix that up?

Reply:
We [have](https://github.com/alcionai/corso/blob/e3363aaa46ca57c5081fd425da3c071d37d95173/src/cli/config/config.go#L77), but I think here we are tying up a configFile path as well for test.

Reply:
I'm not sure I follow that explanation